src/services/sessions.js

```
@@ -1411,6 +1531,24 @@ module.exports = class SessionsHelper {
 				})
 			}
 
+			// Check accessibility after full processing
+			if (userId !== '' && isAMentor !== '') {
+				let isAccessible = await this.checkIfSessionIsAccessible(
+					processDbResponse,
+					userId,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)